airsonic-main/src/main/webapp/WEB-INF/jsp/albumMain.jsp

```diff
@@ -262,6 +262,11 @@
         <span class="header"><a target="_blank" href="${wikipediaUrl}">Wikipedia</a></span> |
         <span class="header"><a target="_blank" href="${allmusicUrl}">allmusic</a></span> |
         <span class="header"><a target="_blank" href="${lastFmUrl}">Last.fm</a></span> |
+        <c:if test="${not empty model.mbReleaseId}">
+          <sub:url value="https://musicbrainz.org/release/${model.mbReleaseId}" var="musicBrainzUrl" encoding="UTF-8">
```

Comment:
model.mbReleaseId needs to be updated to the new variable as well, right?

Reply:
Yup, that should have been the case :)